liblocation/src/main/java/com/mapbox/services/android/core/location/LocationEngine.java

```diff
@@ -115,7 +114,7 @@ public int getPriority() {
    * @param priority One of the {@link LocationEnginePriority}s listed.
```

Comment:
We should include `@LocationEnginePriority.PowerMode` in 👇
https://github.com/mapbox/mapbox-events-android/blob/1b22d627811c739c3a0da6f277756417a5fbfc81/liblocation/src/main/java/com/mapbox/services/android/core/location/LocationEngine.java#L26
https://github.com/mapbox/mapbox-events-android/blob/1b22d627811c739c3a0da6f277756417a5fbfc81/liblocation/src/main/java/com/mapbox/services/android/core/location/LocationEngine.java#L105